modules/apps/map/map-common/test/liferay/MapBase.es.js

```diff
@@ -348,7 +358,12 @@ describe('MapBase', () => {
 		it('stores the given position as position', () => {
 			mapImpl._initializeMap(position);
 
-			expect(position).toBe(mapImpl.position);
+			expect(position.location).toEqual(
+				expect.objectContaining({
+					lat: expect.any(Number),
+					lng: expect.any(Number),
+				})
+			);
```

Comment:
This is also a smell here, when we refactor code we should expect the tests to pass in exactly the same manner. Since you are updating the tests, it makes me suspect that it no longer behaves the same as the original